main.py

```diff
@@ -19,6 +19,24 @@ def validation_credential(user_id: str, user_pin: str) -> bool:
         return False
 
 
+def checking_new_pin(user: User) -> str:
```

Comment:
and later I think I should add some varification for USER_PIN
like : it must be int and the length must be equal to 4